apps/backend/src/workflowEngine/index.ts

```diff
@@ -1,22 +1,22 @@
 import { logger } from '@user-office-software/duo-logger';
 import { container } from 'tsyringe';
 
-import CallDataSource from '../datasources/postgres/CallDataSource';
-import ProposalDataSource from '../datasources/postgres/ProposalDataSource';
-import ProposalSettingsDataSource from '../datasources/postgres/ProposalSettingsDataSource';
+import { Tokens } from '../config/Tokens';
+import { CallDataSource } from '../datasources/CallDataSource';
 import { ProposalEventsRecord } from '../datasources/postgres/records';
+import { ProposalDataSource } from '../datasources/ProposalDataSource';
+import { ProposalSettingsDataSource } from '../datasources/ProposalSettingsDataSource';
 import { Event } from '../events/event.enum';
 import { Proposal } from '../models/Proposal';
 import { StatusChangingEvent } from '../models/StatusChangingEvent';
 import { statusActionEngine } from '../statusActionEngine';
 
-const proposalSettingsDataSource = container.resolve(
-  ProposalSettingsDataSource
-);
-const proposalDataSource = container.resolve(ProposalDataSource);
-const callDataSource = container.resolve(CallDataSource);
-
 const getProposalWorkflowByCallId = (callId: number) => {
+  const proposalSettingsDataSource =
```

Comment:
This file is imported within configure environments files meaning they could not be resolved at the top of this file as the tokens had not yet be registered to the data sources. It would beforehand as the data sources used with the Postgres ones and none of them had a dependency on another dependency that needed injecting.